docs/sql-migration-guide.md

```diff
@@ -218,6 +218,8 @@ license: |
 
   - Since Spark 3.0, the `size` function returns `NULL` for the `NULL` input. In Spark version 2.4 and earlier, this function gives `-1` for the same input. To restore the behavior before Spark 3.0, you can set `spark.sql.legacy.sizeOfNull` to `true`.
 
+  - Since Spark 3.0, the interval literal syntax does not allow multiple unit TO unit statements anymore. For example, `SELECT INTERVAL '1-1' YEAR TO MONTH '2-2' YEAR TO MONTH'` throws parser exception.
```

Comment:
super nit: I think its a little difficult to understand word boundaries "... multiple unit TO unit statements anymore.", so how about `multiple unit-TO-unit statements anymore` or `multiple "unit TO unit" statements anymore`